website/docs/r/organization.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: organization"
+sidebar_current: "docs-mongodbatlas-resource-organization"
+description: |-
+    Provides a Organization resource.
+---
+
+# Resource: mongodbatlas_organization
+
+`mongodbatlas_organization` provides a Organization resource. This allows organization to be created.
+
+~> **IMPORTANT NOTE:**  Creating an organization with this resource creates initial public and private key values that will need to be used in a separate terraform module/blueprint to permit access to new organization.
+
+
+## Example Usage
+
+```terraform
+resource "mongodbatlas_organization" "test" {
+  org_owner_id = "6205e5fffff79cde6f"
+  name = "testCreateORG"
+  description = "test API key from Org Creation Test"
+  role_names = ["ORG_OWNER"]
+}
+```
+
+## Argument Reference
+
+* `name` - (Required) The name of the organization you want to create. (Cannot be changed via this Provider after creation.)
+* `org_owner_id` - (Required) Unique 24-hexadecimal digit string that identifies the Atlas user that you want to assign the Organization Owner role. This user must be a member of the same organization as the calling API key. This is only required when authenticating with Programmatic API Keys.
```

Comment:
```suggestion
* `org_owner_id` - (Required) Unique 24-hexadecimal digit string that identifies the Atlas user that you want to assign the Organization Owner role. This user must be a member of the same organization as the calling API key.  This is only required when authenticating with Programmatic API Keys.  [https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/MongoDB-Cloud-Users/operation/getUserByUsername](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/MongoDB-Cloud-Users/operation/getUserByUsername)
```

FYI @Zuhairahmed - another reason we need Atlas Users added soon. Right now you can't use a data source to easily get the Atlas user id within the provider.

Reply:
makes sense @themantissa, this should be part of near term future release